dummyserver/server.py

```diff
@@ -237,8 +238,12 @@ def run_tornado_app(app, io_loop, certs, scheme, host):
     else:
         http_server = tornado.httpserver.HTTPServer(app)
 
-    sockets = bind_sockets(None, address=host)
-    port = sockets[0].getsockname()[1]
+    if host[:1] == '/':
```

Comment:
[:1] is the same as [0] so let's use the latter for better readability

Reply:
I agree they don't have the same behavior in general, but isn't the type of the `host` parameter in Python 3 a `str` not `bytes` here and empty strings invalid hosts anyways?